GpsDataAnalyzer/fileparser/fileparser.py

```diff
@@ -187,7 +174,103 @@ def parse_xml(self, filename) -> GpsDataSet:
+        # TODO(ameles) replace with variable file path
+        with open(filename, 'r') as csvfile:
+            csvreader = csv.reader(csvfile, delimiter=",")
+            simulation_list = []
+            next_line = next(csvreader)
+            # each simulation starts with a blank line in the csv file
+            while next_line == []:
+                current_simulation = {}
+                try:
+                    metadata_headers = next(csvreader)
+                    metadata_values = next(csvreader)
+                    current_simulation["metadata"] = dict(zip(metadata_headers,metadata_values))
+                    next_line = next(csvreader)  
+                    # start of data for dynamic or end of simulation info for static 
+                    data_dicts = None
+                    if next_line:
+                        data_headers = next_line
+                        next_line = next(csvreader)
+                        current_simulation["data"] = []
+                        while next_line:
+                            # append a dictionary like {'time_from_zero': '2.9', 'x': '-2694168.947', 'y': '-4297224.127', 'z': '3854334.722'}
+                            current_simulation["data"].append(dict(zip(data_headers,next_line)))
+                            next_line = next(csvreader)
+                    simulation_list.append(current_simulation)
+                except StopIteration:
+                    simulation_list.append(current_simulation)
+                    break
+        gps_data_sets = []
+        for simulation in simulation_list:
+            # create GpsMetaData object
+            device = simulation["metadata"]["simulation_type"]
+            start_time = self.parse_time(simulation["metadata"]["start_time"])
+            end_time = self.parse_time(simulation["metadata"]["end_time"])
+            gps_metadata = GpsMetaData(device, None, None, None, start_time, end_time)
+
+            if "data" in simulation:
+                gps_data_list = self.get_dynamic_simulation_data(simulation)
+            else:
+                gps_data_list = self.get_static_simulation_data(simulation)
+
+            data_set = GpsDataSet(gps_metadata, gps_data_list)
+            gps_data_sets.append(data_set)
+        return gps_data_sets
+
+    def get_static_simulation_data(self, simulation):
+        """
+        Reads simulation dictionary and pulls out info for Static Simulation data:
+
+        Args:
+          simulation: a dictionary {"metadata":{....}}
+
+        Returns:
+          A list containing the single data point broadcasted
+        """
+        latitude = float(simulation["metadata"]["latitude"])
+        longitude = float(simulation["metadata"]["longitude"])
+        altitude = 0.0
+        time_stamp = self.parse_time(simulation["metadata"]["start_time"])
+        speed = 0.0
+        new_point = GpsData(latitude, longitude, altitude, speed, time_stamp)
+        return [new_point]
```

Comment:
Could be just `return GpsData(latitude, longitude, altitude, speed, time_stamp) `